src/rapid_pro_to_engagement_db/configuration.py

```diff
@@ -1,20 +1,39 @@
-from dataclasses import dataclass
-from typing import Optional
+from __future__ import annotations
+
+from typing import Dict, Optional
 
 
-@dataclass
 class FlowResultConfiguration:
-    flow_name: str
-    flow_result_field: str
-    engagement_db_dataset: str
+
+    def __init__(self, flow_name: str, flow_result_field: str, engagement_db_dataset: str):
```

Comment:
Can you add a todo to add docstrings for these so they follow the format for the rest of our classes?